### ISSUE TITLE
Add option to sync endpoint default setup python script to disable the certbot container

### ISSUE THREAD

**Issue by wbrunette** [opendatakit/tool-suite-X#264]
If someone wants to supply their own certificate the script should have an option for manually adding a certificate and not start up certbot.

**Comment by sanyud24** [comment]
May I work on this @wbrunette

**Comment by wbrunette** [comment]
@sanyud24 I have assigned the issue to you.

**Comment by sanyud24** [comment]
Hi , I'm little confused about what I've to do when the user chooses "y" in using their own certificate like will they be manually uploading it to the filepath, previously where certbot was storing the certificate or something else . It'll be helpful if I can get some help over how to handle the "y" statement .

**Comment by sanyud24** [comment]
https://github.com/odk-x/sync-endpoint-default-setup/pull/28  is just  a sample PR to know if I'm on the right way.

**Comment by wbrunette** [comment]
@sanyud24 you are definitely on the right track. Here is the documentation issue: https://github.com/odk-x/docs/pull/96

It would be great if you and @khushishikhu coordinated a bit as her ideas should be tested and you are working on the code that could allow that testing :) If you both worked together we could have a complete solution.

**Comment by wbrunette** [comment]
@matt9j feel free to add any additional suggestion or feedback to either pull request.

**Comment by khushishikhu** [comment]
@wbrunette yes sure even I was thinking this too but @sanyud24 were assigned so I let her do that.

**Comment by sanyud24** [comment]
Yeah ohkay , thanks a lot @wbrunette and @khushishikhu

**Comment by khushishikhu** [comment]
> Hi , I'm little confused about what I've to do when the user chooses "y" in using their own certificate like will they be manually uploading it to the filepath, previously where certbot was storing the certificate or something else . It'll be helpful if I can get some help over how to handle the "y" statement .

@sanyud24 if I am understanding your point correctly. https://gist.github.com/chrismclarke/32a0b7331c6027a3e7be4d0d2c0866d3#alt-deploy---via-python-script  Maybe this documentation will help you to take any reference or any other idea

**Comment by sanyud24** [comment]
thanks a lot!! @khushishikhu

**Comment by wbrunette** [comment]
@sanyud24 where is the PR for this one?

**Comment by sanyud24** [comment]
@wbrunette This is the PR - https://github.com/odk-x/sync-endpoint-default-setup/pull/28
I'll be reworking on it though, to fix the issues I couldn't before.
